docs/modelserving/mms/modelmesh/overview.md

```diff
@@ -41,6 +41,7 @@ System-wide configuration parameters can be set by [creating a ConfigMap](https:
 
 - [triton-inference-server](https://github.com/triton-inference-server/server) - NVIDIA's Triton Inference Server
 - [seldon-mlserver](https://github.com/SeldonIO/MLServer) - Python-based inference server
+- [openVINO-model-server](https://github.com/openvinotoolkit/model_server) - OpenVINO Model Server
```

Comment:
Is this runtime installed part of ModelMesh right?

Reply:
yes, it was merged  as part of this PR https://github.com/kserve/modelmesh-serving/pull/141

Reply:
@yuzisun Is there any other concern/question, we'd like to get this merged soon